app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -329,4 +329,8 @@
   <dimen name="promoted_story_card_layout_margin_end">24dp</dimen>
   <dimen name="promoted_story_card_text_size">16sp</dimen>
   <dimen name="promoted_story_card_margin_top">4dp</dimen>
+
+  <!--Topic Practice-->
+  <dimen name="topic_practice_subtopic_layout_margin_start">8dp</dimen>
+
```

Comment:
Do mention `topic_practice_subtopic_layout_margin_end ` also as `0dp` otherwise after merging the files it will end up using the value from `values/dimens.xml` file which is `48dp`

Reply:
Done.